docker-compose.yml

```diff
@@ -1,13 +1,32 @@
 version: '3.8'
 
 services:
+
+  my-mongodb:
+    image: mongo:3.7
+    container_name: mongodb
+    ports:
+      - 27017:27017
+    environment:
+      # MONGO_INITDB_DATABASE: test
+      MONGO_INITDB_ROOT_USERNAME: sdx
+      MONGO_INITDB_ROOT_PASSWORD: passw
```

Comment:
We should not be committing these secrets to the repository.  I left some similar comments in https://github.com/atlanticwave-sdx/sdx-controller/pull/126.

Reply:
This should be fine, it should changed by whomever is deploying the code. And also these will later go into the manifest, which will not be checked into git.